client/src/components/Card.js

```diff
@@ -29,14 +29,25 @@ const DesicionInfo = styled.p`
   font-size: 14px;
 `;
 
-export default function Card() {
+export default function Card({
+  restaurantImg,
+  restaurantName,
+  restaurantDiet,
+  restaurantCuisine,
+}) {
+  const diet = restaurantDiet.toString();
   return (
     <DesicionCardContainer>
-      <DesicionImage src={restaurant} alt="restaurant" />
-      <DesicionHeader>LOREM IPSUM</DesicionHeader>
-      <DesicionInfo>
-        Est duis labore dolor ut consectetur eu do ex anim tempor.
-      </DesicionInfo>
+      <DesicionImage src={restaurantImg} alt="restaurant" />
+      <DesicionHeader>{restaurantName}</DesicionHeader>
+      <DesicionInfo>{`${diet}, ${restaurantCuisine}`}</DesicionInfo>
     </DesicionCardContainer>
   );
```

Comment:
I would probably pass the whole object and destructure it in here so you have less points of failure.